src/assets/scripts/client/aircraft/AircraftModel.js

```diff
@@ -1785,7 +1798,12 @@ export default class AircraftModel {
 
                 // Here we trigger the initial descent. Once vacating the filed cruise altitude, subsequent loops
                 // will enter the below block because the flight phase will have become 'DESCENT'.
-                return hardRestrictedWaypointModel.altitudeMaximum;
+                if (_isNil(hardRestrictedWaypointModel)) {
+                    return this.fms.getBottomAltitude();
+                }
+                else {
```

Comment:
this `else` in unnecessary.  if we reach the `if` block above it we'll be `returned` so anything after that is considered the `else`

_another fair warning: I'm **really** not a fan of `else`.  logic should be clean with a single `if` and `return` instead of an `if`/`else` whenever possible_